context/app/static/js/components/home/RecentEntities/EntityList.tsx

```diff
@@ -37,8 +37,17 @@ export function EntityList<T extends { uuid: string }>({
           <EntityComponent key={entity.uuid} entity={entity} />
         ))}
         <Box p={2} display="flex" justifyContent="center">
-          <Button component={Link} variant="outlined" href={viewAllLink}>
-            View All {entityName}
+          <Button
+            component={Link}
+            variant="outlined"
+            sx={(theme) => ({
+              border: `1px solid ${theme.palette.primary.lowEmphasis}`,
+              borderRadius: `0.25rem`,
```

Comment:
Not sure why husky lint didn't pick it up; I experienced some issues where the TS server/ESLint need to be restarted when swapping back and forth from this branch to others, so it may have been due to that. I fixed the quote types either way 👍🏻